translate/storage/test_mozilla_lang.py

```diff
@@ -38,6 +40,10 @@ def test_comments(self):
         unit.addnote("Hello")
         assert str(unit).find("Hello") == 2
         assert str(unit).find("# Hello") == 0
+        unit.addnote("# TAG: goodbye")
```

Comment:
i put together because the behaviour changed - and i felt like it should go here - ive updated the comment